### ISSUE TITLE
Add a CONTRIBUTING.md file or Contributing section in the README

### ISSUE THREAD

**Issue by theoriginalstove** [xanzy/go-gitlab#1643]
Thanks for this awesome project! I was thinking some Contributing guidelines might be helpful for folks looking to start contributing. Things like if `gofumpt` should be used for formatting. I'm happy to help with this, just need a little direction.

**Comment by svanharmelen** [comment]
Yeah maybe that isn't a bad idea, just never got to it... If you are willing to get me started by creating a PR for an initial doc, then I will tweak it to my liking (i needed) and merge it.

**Comment by theoriginalstove** [comment]
I'll get to it then!
